searx/templates/oscar/search.html

```diff
@@ -6,7 +6,7 @@
         <input type="search" autofocus name="q" class="form-control" id="q" placeholder="{{ _('Search for...') }}" aria-label="{{ _('Search for...') }}" autocomplete="off" value="{{ q }}" accesskey="s">
         <span class="input-group-btn">
             <button type="submit" class="btn btn-default" aria-label="{{ _('Start search') }}"><span class="hide_if_nojs">{{ icon('search') }}</span><span class="hidden active_if_nojs">{{ _('Start search') }}</span></button>
-            <button type="reset" class="btn btn-default" aria-label="{{ _('Clear search') }}"><span class="hide_if_nojs">{{ icon('remove') }}</span><span class="hidden active_if_nojs">{{ _('Clear') }}</span></button>
+            <button type="button" id="clear_search" class="btn btn-default" aria-label="{{ _('Clear search') }}"><span class="hide_if_nojs">{{ icon('remove') }}</span><span class="hidden active_if_nojs">{{ _('Clear') }}</span></button>
```

Comment:
Can you remove the `active_if_nojs` class since the button requires Javascript ?

Reply:
thanks @dalf ! now I understand better those files in my pull request... ;-) 
I made changes you asked (hope so), it that ok ?

Reply:
I've suggested the change without actually checking, and I have messed up... Sorry.

This issue is when Javascript is disabled (in Firefox:  go to `about:config`, set `javascript.enabled` to False) : the button is still shown but empty.

Reply:
replace line 9 by:
```html
<button type="button" id="clear_search" class="btn btn-default hide_if_nojs" aria-label="{{ _('Clear search') }}">{{ icon('remove') }}</button>
```

tested this time.

Reply:
that's really kind of you, thanks !